### ISSUE TITLE
Unable to unlock user account

### ISSUE THREAD

**Issue by thorzinaung** [apache/cloudstack#7585]
##### User Management Issue

 * Improvement Request

##### Management-Server-component

~~~

~~~

##### 4.18.0.0

~~~

~~~

##### SUMMARY

I got locked user account when i wrongly typed password 3 times. I already searched in root privileges and cannot find to unlock account.  

##### STEPS TO REPRODUCE

~~~
<img width="782" alt="image" src="https://github.com/apache/cloudstack/assets/122773216/b29fda4a-e46b-42bb-afb1-642a93272267">

~~~

##### EXPECTED RESULTS

~~~

~~~

##### ACTUAL RESULTS

~~~

~~~

**Comment by thorzinaung** [comment]
<img width="782" alt="image" src="https://github.com/apache/cloudstack/assets/122773216/1bb1350f-6315-4865-9184-2ac3f799ce12">

**Comment by kiranchavala** [comment]
@thorzinaung 

I am not able to reproduce the issue in my lab after typing the password incorrectly 3 times

What type of role you assigned for (angyekyaw) when creating the account ?

Is ur system linked to any LDAP integration ?


<img width="518" alt="Screenshot 2023-06-05 at 12 51 43 PM" src="https://github.com/apache/cloudstack/assets/1401014/d1d110ee-3694-46c9-aee8-6abe41b7f7a6">


Can you login as Admin user > Account > View Users  Check the status of the user


<img width="1181" alt="Screenshot 2023-06-05 at 12 56 49 PM" src="https://github.com/apache/cloudstack/assets/1401014/39b6ef8e-36ff-48a1-a82b-faad2c53cc6a">

<img width="503" alt="Screenshot 2023-06-05 at 12 57 02 PM" src="https://github.com/apache/cloudstack/assets/1401014/6afbf53a-07c5-4d66-b56a-4599b858bc1b">


You can try the enable user api call 

https://cloudstack.apache.org/api/apidocs-4.18/apis/enableUser.html

**Comment by kiranchavala** [comment]
@thorzinaung , please if it's showing green light for the User setting

Admin user > Select  Account > Select the user under the account > View Users> Check the status of the user

Or execute the list user api call and check the state

https://cloudstack.apache.org/api/apidocs-4.18/apis/listUsers.html

> list users username=kiranuser listall=true
{
  "count": 1,
  "user": [
    {
      "account": "kiranuser",
      "accountid": "ca73fc64-586e-4d06-9596-f4b2cf449a92",
      "accounttype": 0,
      "created": "2023-06-05T07:19:46+0000",
      "domain": "ROOT",
      "domainid": "0627bb29-ea34-11ed-aeb5-1e0092000276",
      "email": "kiranchavala@gfsgsal.com",
      "firstname": "kiran",
      "id": "08749f72-6ccc-4e2f-a8ff-19e3a885b187",
      "is2faenabled": false,
      "is2famandated": false,
      "iscallerchilddomain": false,
      "isdefault": false,
      "lastname": "user",
      "roleid": "1445ab81-ea34-11ed-aeb5-1e0092000276",
      "rolename": "User",
      "roletype": "User",
      "**state": "disabled",**
      "username": "kiranuser",
      "usersource": "native"
    }
  ]
}



Could you please try to clear the browser cache or try in an incognito window of the browser

Also, please  provide the complete management server logs

**Comment by thorzinaung** [comment]
yes. @kiranchavala thank you. Also, i'm newbie to ACS. how can i take public ip to instance like Elastic ip in aws.

**Comment by rajujith** [comment]
@thorzinaung Assuming that you are using an isolated guest network, you could click on the instance and go to its details. Select the tab NICs, and click on the network it is attached to. Click on the tab Public IP addresses. Here you can either acquire a new public IP and configure a static NAT or you could even configure a port forwarding rule on the source NAT IP itself. Make sure to allow the ports and subnets on the firewall tab on the IP. 

Some of these are explained here : 
http://docs.cloudstack.apache.org/en/latest/adminguide/networking_and_traffic.html#enabling-or-disabling-static-nat
https://www.shapeblue.com/apache-cloudstack-poc-guide/#tab-id-3

**Comment by rohityadavcloud** [comment]
@thorzinaung is the issue that even admins can't unlock the account?
